mpact/services.py

```diff
@@ -235,16 +235,17 @@ async def get_chat_msg(phone, chat_id, limit, offset):
                     extract_messages(message, msgs)
 
             return {
-                DATA: {"messages": msgs, IS_SUCCESS: True},
+                DATA: {"messages": msgs[::-1], IS_SUCCESS: True},
                 STATUS: status.HTTP_200_OK,
             }
         return NOT_AUTHORIZED
 
 
 def extract_messages(message, msgs):
-    msg = {}
-    msg["id"] = message.id
-    msg["sender"] = message.sender.first_name
-    msg[MESSAGE] = message.text
-    msg["date"] = message.date
-    msgs.append(msg)
+    if message.message:
+        msg = {}
+        msg["id"] = message.id
+        msg["sender"] = message.sender.first_name
+        msg[MESSAGE] = message.text
+        msg["date"] = message.date
+        msgs.append(msg)
```

Comment:
This feels a little awkward. How about this instead?
```suggestion
        msgs.append({
            "id": message.id,
            "sender": message.sender.first_name,
            MESSAGE: message.text,
            "date": message.date,
        })
```